### DISCUSSION TITLE
Does `guild.commands.set` affect the global rate limit?

### DISCUSSION THREAD

**Discussion by Wondermarin** [discordjs/discord.js#7047] [Q&A] [answered]
The official Discord documentation:

> There is a global rate limit of 200 application command creates per day, per guild

Does `guild.commands.set` affect this limit?
And how does the limit behave if `guild.commands.set` is used to update commands, not to create them.

**Comment by Jarco-dev** [comment, marked as answer]
Updating commands doesn't count, creating a command does. (even with set it applies in the same way)
